ember-welcome-page/src/components/welcome-page.ts

```diff
@@ -0,0 +1,34 @@
+import { getOwner } from '@ember/application';
+/* @ts-expect-error: Cannot find module '@ember/version' or its corresponding type declarations. */
+import { VERSION } from '@ember/version';
+import Component from '@glimmer/component';
+import './welcome-page.css';
+
+function isLatestVersion(): boolean {
+  const stableRegex = /^\d+\.\d+\.\d+$/;
+
+  return !stableRegex.test(VERSION as string);
+}
+
+export default class WelcomePageComponent extends Component {
+  get rootURL(): string {
+    const config = getOwner(this).factoryFor('config:environment');
+
+    if (config) {
+      return (config.class as unknown as { rootURL: string }).rootURL;
+    }
+
+    return '/';
+  }
+
+  get urlForEmberGuides(): string {
+    if (isLatestVersion()) {
+      return `https://guides.emberjs.com/release`;
```

Comment:
I just noticed, the URL https://guides.emberjs.com/current/tutorial/ember-cli/ may be outdated. The link redirects the user to https://guides.emberjs.com/release/tutorial/part-1/orientation/ (note, it's now `/release`, not `/current`).

In the template, I sent the user to `https://guides.emberjs.com/release/tutorial` and let the Ember Guides redirect the user to the first page in the tutorial.